### ISSUE TITLE
error generated when packer is run in azure devops ci/cd build pipeline

### ISSUE THREAD

**Issue by alldayanyday** [hashicorp/packer#7233]
When running packer as part of an azure devops build pipeline, packer builds the virtual machine successfully, but when the shutdown command is run and the packer attempts to remove the floppy drive:

```
2019-01-27T22:14:17.4498027Z     virtualbox-iso: Removing floppy drive...
2019-01-27T22:14:17.5217620Z ==> virtualbox-iso: Error removing floppy: VBoxManage error: VBoxManage.exe: error: Failed to get a console object from the direct session (VBOX_E_INVALID_OBJECT_STATE)
2019-01-27T22:14:17.5219963Z ==> virtualbox-iso: VBoxManage.exe: error: Details: code VBOX_E_VM_ERROR (0x80bb0003), component MachineWrap, interface IMachine, callee IUnknown
2019-01-27T22:14:17.5222850Z ==> virtualbox-iso: VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Shared)" at line 336 of file VBoxManageStorageController.cpp
2019-01-27T22:14:17.7045588Z ==> virtualbox-iso: Deregistering and deleting VM...
2019-01-27T22:14:18.7089097Z ==> virtualbox-iso: Deleting output directory...
2019-01-27T22:14:18.7193504Z Build 'virtualbox-iso' errored: Error removing floppy: VBoxManage error: VBoxManage.exe: error: Failed to get a console object from the direct session (VBOX_E_INVALID_OBJECT_STATE)
2019-01-27T22:14:18.7196339Z VBoxManage.exe: error: Details: code VBOX_E_VM_ERROR (0x80bb0003), component MachineWrap, interface IMachine, callee IUnknown
2019-01-27T22:14:18.7198789Z VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Shared)" at line 336 of file VBoxManageStorageController.cpp
2019-01-27T22:14:18.7212410Z 
2019-01-27T22:14:18.7214245Z ==> Some builds didn't complete successfully and had errors:
2019-01-27T22:14:18.7218684Z --> virtualbox-iso: Error removing floppy: VBoxManage error: VBoxManage.exe: error: Failed to get a console object from the direct session (VBOX_E_INVALID_OBJECT_STATE)
2019-01-27T22:14:18.7221861Z VBoxManage.exe: error: Details: code VBOX_E_VM_ERROR (0x80bb0003), component MachineWrap, interface IMachine, callee IUnknown
2019-01-27T22:14:18.7224483Z VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Shared)" at line 336 of file VBoxManageStorageController.cpp
```

## The below is the full console output from one of the failed builds:

```
2019-01-27T19:51:39.7433650Z ##[section]Starting: PowerShell Script
2019-01-27T19:51:39.7439530Z ==============================================================================
2019-01-27T19:51:39.7439650Z Task         : PowerShell
2019-01-27T19:51:39.7439705Z Description  : Run a PowerShell script on Windows, macOS, or Linux.
2019-01-27T19:51:39.7439761Z Version      : 2.140.2
2019-01-27T19:51:39.7439825Z Author       : Microsoft Corporation
2019-01-27T19:51:39.7439881Z Help         : [More Information](https://go.microsoft.com/fwlink/?LinkID=613736)
2019-01-27T19:51:39.7439932Z ==============================================================================
2019-01-27T19:51:48.3953960Z Generating script.
2019-01-27T19:51:48.3965677Z Formatted command: . 'C:\Users\whiggs\Desktop\vsts\_work\1\s\build.ps1'
2019-01-27T19:51:48.4710616Z ##[command]"C:\windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'C:\Users\whiggs\Desktop\vsts\_work\_temp\c9bb4841-aa48-4b78-84fc-a47d69071622.ps1'"
2019-01-27T19:51:55.8649470Z 
2019-01-27T19:51:55.8653960Z Name                     Version          DynamicOptions                                                               
2019-01-27T19:51:55.8656735Z ----                     -------          --------------                                                               
2019-01-27T19:51:55.8681586Z NuGet                    3.0.0.1          Destination, ExcludeVersion, Scope, SkipDependencies, Headers, FilterOnTag...
2019-01-27T19:52:13.0486301Z WARNING: The version '1.2.4' of module 'PackageManagement' is currently in use. Retry the operation after closing the 
2019-01-27T19:52:13.0486814Z applications.
2019-01-27T19:52:16.8598212Z WARNING: The version '1.2.4' of module 'PackageManagement' is currently in use. Retry the operation after closing the 
2019-01-27T19:52:16.8598738Z applications.
2019-01-27T19:53:08.9132655Z Executing buildpacker
2019-01-27T19:53:08.9926213Z 
2019-01-27T19:53:15.4840343Z virtualbox-iso output will be in this color.
2019-01-27T19:53:15.4840616Z 
2019-01-27T19:53:16.7514488Z ==> virtualbox-iso: Retrieving Guest additions
2019-01-27T19:53:16.7522455Z     virtualbox-iso: Using file in-place: file:///C:/Program%20Files/Oracle/VirtualBox/VBoxGuestAdditions.iso
2019-01-27T19:53:16.7524653Z ==> virtualbox-iso: Retrieving ISO
2019-01-27T19:53:44.4235548Z     virtualbox-iso: Using file in-place: file://C:/iso/SW_DVD9_Win_Svr_STD_Core_and_DataCtr_Core_2016_64Bit_English_-2_MLF_X21-22843.ISO
2019-01-27T19:53:44.4267626Z ==> virtualbox-iso: Creating floppy disk...
2019-01-27T19:53:44.4300290Z     virtualbox-iso: Copying files flatly from floppy_files
2019-01-27T19:53:44.4302497Z     virtualbox-iso: Copying file: ./misc/run.ps1
2019-01-27T19:53:44.4312256Z     virtualbox-iso: Copying file: ./misc/uac.inf
2019-01-27T19:53:44.4330216Z     virtualbox-iso: Copying file: ./misc/vmwaretools.bat
2019-01-27T19:53:44.4338072Z     virtualbox-iso: Copying file: ./misc/virtualtools.bat
2019-01-27T19:53:44.4344355Z     virtualbox-iso: Copying file: ./misc/baseline.inf
2019-01-27T19:53:44.4349795Z     virtualbox-iso: Copying file: ./answer_files/server2016/Autounattend.xml
2019-01-27T19:53:44.4356321Z     virtualbox-iso: Copying file: ./misc/pack_shutdown.bat
2019-01-27T19:53:44.4361388Z     virtualbox-iso: Done copying files from floppy_files
2019-01-27T19:53:44.4363086Z     virtualbox-iso: Collecting paths from floppy_dirs
2019-01-27T19:53:44.4365425Z     virtualbox-iso: Resulting paths from floppy_dirs : []
2019-01-27T19:53:44.4367195Z     virtualbox-iso: Done copying paths from floppy_dirs
2019-01-27T19:53:44.8846461Z ==> virtualbox-iso: Creating virtual machine...
2019-01-27T19:53:45.1354478Z ==> virtualbox-iso: Creating hard drive...
2019-01-27T19:53:45.4070580Z ==> virtualbox-iso: Attaching floppy disk...
2019-01-27T19:53:45.4979837Z ==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 3613)
2019-01-27T19:53:45.5362717Z ==> virtualbox-iso: Executing custom VBoxManage commands...
2019-01-27T19:53:45.5366510Z     virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1548618795 --memory 4096
2019-01-27T19:53:45.5745418Z     virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1548618795 --cpus 2
2019-01-27T19:53:45.6102135Z ==> virtualbox-iso: Starting the virtual machine...
2019-01-27T19:53:51.5722004Z ==> virtualbox-iso: Waiting 10s for boot...
2019-01-27T19:54:01.5748172Z ==> virtualbox-iso: Typing the boot command...
2019-01-27T19:54:01.5757127Z ==> virtualbox-iso: Using winrm communicator to connect: 127.0.0.1
2019-01-27T19:54:01.5763965Z ==> virtualbox-iso: Waiting for WinRM to become available...
2019-01-27T22:13:51.8840412Z     virtualbox-iso: WinRM connected.
2019-01-27T22:13:51.8850462Z     virtualbox-iso: #< CLIXML
2019-01-27T22:13:51.8850723Z     virtualbox-iso: <Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><Obj S="progress" RefId="0"><TN RefId="0"><T>System.Management.Automation.PSCustomObject</T><T>System.Object</T></TN><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj><Obj S="progress" RefId="1"><TNRef RefId="0" /><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj></Objs>
2019-01-27T22:13:51.8854572Z ==> virtualbox-iso: Connected to WinRM!
2019-01-27T22:13:51.9443039Z ==> virtualbox-iso: Uploading VirtualBox version info (6.0.2)
2019-01-27T22:14:02.6296080Z ==> virtualbox-iso: Gracefully halting virtual machine...
2019-01-27T22:14:17.4498027Z     virtualbox-iso: Removing floppy drive...
2019-01-27T22:14:17.5217620Z ==> virtualbox-iso: Error removing floppy: VBoxManage error: VBoxManage.exe: error: Failed to get a console object from the direct session (VBOX_E_INVALID_OBJECT_STATE)
2019-01-27T22:14:17.5219963Z ==> virtualbox-iso: VBoxManage.exe: error: Details: code VBOX_E_VM_ERROR (0x80bb0003), component MachineWrap, interface IMachine, callee IUnknown
2019-01-27T22:14:17.5222850Z ==> virtualbox-iso: VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Shared)" at line 336 of file VBoxManageStorageController.cpp
2019-01-27T22:14:17.7045588Z ==> virtualbox-iso: Deregistering and deleting VM...
2019-01-27T22:14:18.7089097Z ==> virtualbox-iso: Deleting output directory...
2019-01-27T22:14:18.7193504Z Build 'virtualbox-iso' errored: Error removing floppy: VBoxManage error: VBoxManage.exe: error: Failed to get a console object from the direct session (VBOX_E_INVALID_OBJECT_STATE)
2019-01-27T22:14:18.7196339Z VBoxManage.exe: error: Details: code VBOX_E_VM_ERROR (0x80bb0003), component MachineWrap, interface IMachine, callee IUnknown
2019-01-27T22:14:18.7198789Z VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Shared)" at line 336 of file VBoxManageStorageController.cpp
2019-01-27T22:14:18.7212410Z 
2019-01-27T22:14:18.7214245Z ==> Some builds didn't complete successfully and had errors:
2019-01-27T22:14:18.7218684Z --> virtualbox-iso: Error removing floppy: VBoxManage error: VBoxManage.exe: error: Failed to get a console object from the direct session (VBOX_E_INVALID_OBJECT_STATE)
2019-01-27T22:14:18.7221861Z VBoxManage.exe: error: Details: code VBOX_E_VM_ERROR (0x80bb0003), component MachineWrap, interface IMachine, callee IUnknown
2019-01-27T22:14:18.7224483Z VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Shared)" at line 336 of file VBoxManageStorageController.cpp
2019-01-27T22:14:18.7224641Z 
2019-01-27T22:14:18.7225723Z ==> Builds finished but no artifacts were created.
2019-01-27T22:14:19.7158176Z Error: 1/27/2019 5:14:19 PM: 
2019-01-27T22:14:19.7158492Z At C:\Program Files (x86)\WindowsPowerShell\Modules\Psake\4.7.4\public\Exec.ps1:79 char:17
2019-01-27T22:14:19.7158714Z  +                 throw "Exec: $errorMessage"
2019-01-27T22:14:19.7159262Z  +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~ [<<==>>] Exception: Exec: Error executing command  packer build -var "version=$newver" -var "vagrantkey=$env:vagrant_APIkey" -force -only virtualbox-iso (Join-Path $baseDir jsonformatter.json) .
2019-01-27T22:14:19.7929924Z 
2019-01-27T22:14:19.7930193Z 
2019-01-27T22:14:20.4172083Z ##[error]PowerShell exited with code '1'.
2019-01-27T22:14:20.4599671Z ##[section]Finishing: PowerShell Script

```

## The below is one of the successful builds with no major differences between the two builds:

```
2019-01-24T04:23:42.0965110Z ##[section]Starting: PowerShell Script
2019-01-24T04:23:42.0969891Z ==============================================================================
2019-01-24T04:23:42.0969989Z Task         : PowerShell
2019-01-24T04:23:42.0970040Z Description  : Run a PowerShell script on Windows, macOS, or Linux.
2019-01-24T04:23:42.0970092Z Version      : 2.140.2
2019-01-24T04:23:42.0970139Z Author       : Microsoft Corporation
2019-01-24T04:23:42.0970199Z Help         : [More Information](https://go.microsoft.com/fwlink/?LinkID=613736)
2019-01-24T04:23:42.0970250Z ==============================================================================
2019-01-24T04:23:50.2400966Z Generating script.
2019-01-24T04:23:50.2413795Z Formatted command: . 'C:\Users\whiggs\Desktop\vsts\_work\1\s\build.ps1'
2019-01-24T04:23:50.2970604Z ##[command]"C:\windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'C:\Users\whiggs\Desktop\vsts\_work\_temp\8e97809c-76d2-407d-8f96-461cff7969d9.ps1'"
2019-01-24T04:23:57.6516711Z 
2019-01-24T04:23:57.6529973Z Name                     Version          DynamicOptions                                                               
2019-01-24T04:23:57.6531799Z ----                     -------          --------------                                                               
2019-01-24T04:23:57.6549153Z NuGet                    3.0.0.1          Destination, ExcludeVersion, Scope, SkipDependencies, Headers, FilterOnTag...
2019-01-24T04:24:14.4557616Z WARNING: The version '1.2.4' of module 'PackageManagement' is currently in use. Retry the operation after closing the 
2019-01-24T04:24:14.4558198Z applications.
2019-01-24T04:24:18.0785272Z WARNING: The version '1.2.4' of module 'PackageManagement' is currently in use. Retry the operation after closing the 
2019-01-24T04:24:18.0785648Z applications.
2019-01-24T04:25:08.9701355Z Executing buildpacker
2019-01-24T04:25:09.0439763Z 
2019-01-24T04:25:15.6125013Z virtualbox-iso output will be in this color.
2019-01-24T04:25:15.6125445Z 
2019-01-24T04:25:16.2408168Z ==> virtualbox-iso: Retrieving Guest additions
2019-01-24T04:25:16.2430816Z     virtualbox-iso: Using file in-place: file:///C:/Program%20Files/Oracle/VirtualBox/VBoxGuestAdditions.iso
2019-01-24T04:25:16.2433130Z ==> virtualbox-iso: Retrieving ISO
2019-01-24T04:25:42.8092768Z     virtualbox-iso: Using file in-place: file://C:/iso/SW_DVD9_Win_Svr_STD_Core_and_DataCtr_Core_2016_64Bit_English_-2_MLF_X21-22843.ISO
2019-01-24T04:25:42.8093285Z ==> virtualbox-iso: Creating floppy disk...
2019-01-24T04:25:42.8093525Z     virtualbox-iso: Copying files flatly from floppy_files
2019-01-24T04:25:42.8093974Z     virtualbox-iso: Copying file: ./misc/run.ps1
2019-01-24T04:25:42.8094154Z     virtualbox-iso: Copying file: ./misc/uac.inf
2019-01-24T04:25:42.8094320Z     virtualbox-iso: Copying file: ./misc/vmwaretools.bat
2019-01-24T04:25:42.8094479Z     virtualbox-iso: Copying file: ./misc/virtualtools.bat
2019-01-24T04:25:42.8094645Z     virtualbox-iso: Copying file: ./misc/baseline.inf
2019-01-24T04:25:42.8094825Z     virtualbox-iso: Copying file: ./answer_files/server2016/Autounattend.xml
2019-01-24T04:25:42.8094940Z     virtualbox-iso: Copying file: ./misc/pack_shutdown.bat
2019-01-24T04:25:42.8095086Z     virtualbox-iso: Done copying files from floppy_files
2019-01-24T04:25:42.8095236Z     virtualbox-iso: Collecting paths from floppy_dirs
2019-01-24T04:25:42.8095364Z     virtualbox-iso: Resulting paths from floppy_dirs : []
2019-01-24T04:25:42.8095638Z     virtualbox-iso: Done copying paths from floppy_dirs
2019-01-24T04:25:43.2050267Z ==> virtualbox-iso: Creating virtual machine...
2019-01-24T04:25:43.4257738Z ==> virtualbox-iso: Creating hard drive...
2019-01-24T04:25:43.6709019Z ==> virtualbox-iso: Attaching floppy disk...
2019-01-24T04:25:43.7517083Z ==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 3969)
2019-01-24T04:25:43.7910680Z ==> virtualbox-iso: Executing custom VBoxManage commands...
2019-01-24T04:25:43.7915739Z     virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1548303915 --memory 4096
2019-01-24T04:25:43.8338776Z     virtualbox-iso: Executing: modifyvm packer-virtualbox-iso-1548303915 --cpus 2
2019-01-24T04:25:43.8741045Z ==> virtualbox-iso: Starting the virtual machine...
2019-01-24T04:25:49.3087335Z ==> virtualbox-iso: Waiting 10s for boot...
2019-01-24T04:25:59.3100963Z ==> virtualbox-iso: Typing the boot command...
2019-01-24T04:25:59.3104085Z ==> virtualbox-iso: Using winrm communicator to connect: 127.0.0.1
2019-01-24T04:25:59.3105984Z ==> virtualbox-iso: Waiting for WinRM to become available...
2019-01-24T06:45:32.0743011Z     virtualbox-iso: WinRM connected.
2019-01-24T06:45:32.0746214Z     virtualbox-iso: #< CLIXML
2019-01-24T06:45:32.0776526Z     virtualbox-iso: <Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><Obj S="progress" RefId="0"><TN RefId="0"><T>System.Management.Automation.PSCustomObject</T><T>System.Object</T></TN><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj><Obj S="progress" RefId="1"><TNRef RefId="0" /><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj></Objs>
2019-01-24T06:45:32.0776971Z ==> virtualbox-iso: Connected to WinRM!
2019-01-24T06:45:32.1280355Z ==> virtualbox-iso: Uploading VirtualBox version info (6.0.2)
2019-01-24T06:45:41.6097973Z ==> virtualbox-iso: Gracefully halting virtual machine...
2019-01-24T06:45:56.3707347Z     virtualbox-iso: Removing floppy drive...
2019-01-24T06:46:12.0240353Z     virtualbox-iso: Removing guest additions drive...
2019-01-24T06:46:13.0585960Z ==> virtualbox-iso: Preparing to export machine...
2019-01-24T06:46:13.0595531Z     virtualbox-iso: Deleting forwarded port mapping for the communicator (SSH, WinRM, etc) (host port 3969)
2019-01-24T06:46:13.1403844Z ==> virtualbox-iso: Exporting virtual machine...
2019-01-24T06:46:13.1413535Z     virtualbox-iso: Executing: export packer-virtualbox-iso-1548303915 --output output-virtualbox-iso\packer-virtualbox-iso-1548303915.ovf
2019-01-24T07:12:11.7791233Z ==> virtualbox-iso: Deregistering and deleting VM...
2019-01-24T07:12:12.8319353Z ==> virtualbox-iso: Running post-processor: vagrant
2019-01-24T07:12:12.8360425Z ==> virtualbox-iso (vagrant): Creating Vagrant box for 'virtualbox' provider
2019-01-24T07:12:12.8395240Z     virtualbox-iso (vagrant): Copying from artifact: output-virtualbox-iso\packer-virtualbox-iso-1548303915-disk001.vmdk
2019-01-24T07:13:09.1730706Z     virtualbox-iso (vagrant): Copying from artifact: output-virtualbox-iso\packer-virtualbox-iso-1548303915.ovf
2019-01-24T07:13:09.1780724Z     virtualbox-iso (vagrant): Renaming the OVF to box.ovf...
2019-01-24T07:13:09.1827778Z     virtualbox-iso (vagrant): Using custom Vagrantfile: vagrantfile.tpl
2019-01-24T07:13:09.1897747Z     virtualbox-iso (vagrant): Compressing: Vagrantfile
2019-01-24T07:13:09.1974458Z     virtualbox-iso (vagrant): Compressing: box.ovf
2019-01-24T07:13:09.1983336Z     virtualbox-iso (vagrant): Compressing: metadata.json
2019-01-24T07:13:09.1993233Z     virtualbox-iso (vagrant): Compressing: packer-virtualbox-iso-1548303915-disk001.vmdk
2019-01-24T07:16:07.1437269Z ==> virtualbox-iso: Running post-processor: vagrant-cloud
2019-01-24T07:16:07.1738959Z ==> virtualbox-iso (vagrant-cloud): Verifying box is accessible: whiggs/mybox
2019-01-24T07:16:07.6560768Z     virtualbox-iso (vagrant-cloud): Box accessible and matches tag
2019-01-24T07:16:07.6562624Z ==> virtualbox-iso (vagrant-cloud): Creating version: 1.7.0
2019-01-24T07:16:07.9520285Z ==> virtualbox-iso (vagrant-cloud): Creating provider: virtualbox
2019-01-24T07:16:08.3065693Z ==> virtualbox-iso (vagrant-cloud): Preparing upload of box: windows2016min-virtualbox.box
2019-01-24T07:16:08.7771681Z ==> virtualbox-iso (vagrant-cloud): Uploading box: windows2016min-virtualbox.box
2019-01-24T07:16:08.7780529Z     virtualbox-iso (vagrant-cloud): Depending on your internet connection and the size of the box,
2019-01-24T07:16:08.7781850Z     virtualbox-iso (vagrant-cloud): this may take some time
2019-01-24T07:16:08.7787624Z     virtualbox-iso (vagrant-cloud): Uploading box, attempt 1
2019-01-24T08:15:45.3094976Z     virtualbox-iso (vagrant-cloud): Box successfully uploaded
2019-01-24T08:15:45.3101609Z ==> virtualbox-iso (vagrant-cloud): Releasing version: 1.7.0
2019-01-24T08:15:45.6730714Z     virtualbox-iso (vagrant-cloud): Version successfully released and available
2019-01-24T08:15:45.6783670Z Build 'virtualbox-iso' finished.
2019-01-24T08:15:45.6790085Z 
2019-01-24T08:15:45.6790825Z ==> Builds finished. The artifacts of successful builds are:
2019-01-24T08:15:45.6811669Z --> virtualbox-iso: 'virtualbox' provider box: windows2016min-virtualbox.box
2019-01-24T08:15:45.6826765Z --> virtualbox-iso: 'virtualbox': whiggs/mybox
2019-01-24T08:15:45.9629928Z 
2019-01-24T08:15:45.9630262Z psake succeeded executing .\psake.ps1
2019-01-24T08:15:45.9630364Z 
2019-01-24T08:15:45.9725474Z ----------------------------------------------------------------------
2019-01-24T08:15:45.9725807Z Build Time Report
2019-01-24T08:15:45.9727522Z ----------------------------------------------------------------------
2019-01-24T08:15:45.9979845Z Name        Duration    
2019-01-24T08:15:45.9981188Z ----        --------    
2019-01-24T08:15:45.9983681Z buildpacker 03:50:37.064
2019-01-24T08:15:45.9985961Z Total:      03:50:37.108
2019-01-24T08:15:46.0101230Z 
2019-01-24T08:15:46.0101431Z 
2019-01-24T08:15:46.1157080Z ##[section]Finishing: PowerShell Script

```
- Packer version from `packer version`
Packer 1.3.3
- Host platform
Windows server 2016 is the operating system of the Azure Devops build server
- The _simplest example template and scripts_ needed to reproduce the bug.
Attached is a zip with the repository's contents.
[imagecapture.zip](https://github.com/hashicorp/packer/files/2801480/imagecapture.zip)

**Comment by SwampDragons** [comment]
oh geez, you're right. I just saw "azure" in the title and categorized too quickly. Thanks for the catch.

**Comment by alldayanyday** [comment]
I apologize for the confusion.  This has nothing to do with the azure builder.  The only reason azure factors into this issue is because the error is generated when packer is running as part of an azure dev ops ci/cd build pipeline.  The azure build pipeline runs a powershell script which calls packer and uses, as @boumenot stated, the virtualbox-iso builder.  However, I was able to figure out what the issue was.  I had not configured the azure devops build pipeline to clear the project directory, so I am assuming that leftover files were caused some kind of conflict, as the error stopped occurring when I configured the pipeline to clear the project directory.  Closing the issue.
